lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -508,10 +515,13 @@ export class AuthorizationCodeClient extends BaseClient {
             request.correlationId
         );
 
-        const parameterBuilder = new RequestParameterBuilder();
+        const parameterBuilder = new RequestParameterBuilder(
+            this.performanceClient
+        );
 
         parameterBuilder.addClientId(
-            request.extraQueryParameters?.[AADServerParamKeys.CLIENT_ID] ||
+            request.brokerParameters?.embeddedClientId ||
+                request.extraQueryParameters?.[AADServerParamKeys.CLIENT_ID] ||
```

Comment:
Do we want to support both `extraQueryParameters` and the first order support? Can we not just stick to first order  ones?

Reply:
Correction: I understand this is a breaking change for hosts, so may be we can add a notice to prefer first order over extraQueryParams.

Reply:
Added a note to `BaseAuthRequest`.